--- Makefile	2006-06-06 19:25:10.000000000 +0200
+++ Makefile-new	2006-07-15 02:33:01.228046632 +0200
@@ -2,7 +2,7 @@
 LDFLAGS := 
 
 DESTDIR :=
-PREFIX := /usr/local
+PREFIX := /usr
 INSTDIR := $(DESTDIR)$(PREFIX)/sbin
 
 MODULES = text fbsplash usplash
@@ -37,7 +37,7 @@
 
 $(INSTDIR)/%: %
 	strip $<
-	install -m 755 -o root -g root $< $@
+	install -m 755 $< $@
 
 install: all $(patsubst %,$(INSTDIR)/%,$(TARGETS))
 
