--- suspend2-userui-0.7.0/Makefile	2007-01-16 23:16:46.000000000 +0100
+++ suspend2-userui-0.7.0/Makefile-new	2007-04-02 23:50:04.583044390 +0200
@@ -2,14 +2,14 @@
 LDFLAGS :=
 
 DESTDIR :=
-PREFIX := /usr/local
+PREFIX := /usr
 INSTDIR := $(DESTDIR)$(PREFIX)/sbin
 
 MODULES = text fbsplash usplash
 TARGETS = $(patsubst %,suspend2ui_%,$(MODULES))
 CORE_OBJECTS = userui_core.o
 
-FBSPLASH_LIBS = -lmng -lpng -ljpeg -lz -lfreetype -llcms -lm
+FBSPLASH_LIBS = -lmng -lpng12 -ljpeg -lz -lfreetype -llcms -lm
 
 USPLASH_LIBS = -ldl
 
@@ -22,13 +22,13 @@
 	make -C $@ all
 
 suspend2ui_text: $(CORE_OBJECTS) userui_text.o
-	$(CC) $(LDFLAGS) -static $^ -o $@
+	$(CC) $(LDFLAGS) $^ -o $@
 
 suspend2ui_fbsplash: $(CORE_OBJECTS) fbsplash/userui_fbsplash.o
-	$(CC) $(LDFLAGS) -static $^ -o $@ $(FBSPLASH_LIBS)
+	$(CC) $(LDFLAGS) $^ -o $@ $(FBSPLASH_LIBS)
 
 suspend2ui_usplash: $(CORE_OBJECTS) usplash/userui_usplash.o
-	$(CC) $(filter-out -static,$(LDFLAGS)) $^ -o $@ $(USPLASH_LIBS)
+	$(CC) $^ -o $@ $(USPLASH_LIBS)
 
 clean:
 	$(RM) *.o $(TARGETS)
@@ -36,8 +36,7 @@
 	make -C usplash clean
 
 $(INSTDIR)/%: %
-	strip $<
-	install -m 755 -o root -g root $< $@
+	install -m 755 $< $@
 
 install: all $(patsubst %,$(INSTDIR)/%,$(TARGETS))
 
