--- suspend2-userui-0.7.2/Makefile.orig	2007-08-23 17:34:13.812277749 +0200
+++ suspend2-userui-0.7.2/Makefile	2007-08-23 17:35:35.045532820 +0200
@@ -5,7 +5,7 @@
 PREFIX := /usr
 INSTDIR := $(DESTDIR)$(PREFIX)/sbin
 
-MODULES = text fbsplash usplash
+MODULES = text
 TARGETS = $(patsubst %,tuxoniceui_%,$(MODULES))
 CORE_OBJECTS = userui_core.o
 
@@ -30,7 +30,7 @@
 	@echo "       - To build this, you must have libusplash0 installed."
 
 all: $(TARGETS)
-
+	
 fbsplash:
 	make -C $@ all
 
