diff -aurN suspend2-userui-0.6.4.orig/usplash/bogl.c suspend2-userui-0.6.4/usplash/bogl.c
--- suspend2-userui-0.6.4.orig/usplash/bogl.c	2006-06-06 19:25:11.000000000 +0200
+++ suspend2-userui-0.6.4/usplash/bogl.c	2006-07-16 09:21:17.867042999 +0200
@@ -31,6 +31,8 @@
 #include <string.h>
 #if defined(__i386__) || defined(__alpha__)
 #include <sys/io.h>
+#else
+#include <asm/io.h>
 #endif
 #include <sys/ioctl.h>
 #include <sys/mman.h>
